www/config.yml

```diff
@@ -277,6 +277,17 @@ chapters:
       location:
             latitude: 55.9531
             longitude: -3.1889
+            
+    
+    - name: Ghana
+      website: ghana.pyladies.com
+      image: pyladies_ghana.jpg
```

Comment:
Can you add the image too? It should be placed in `www/_assets/images/pyladies_ghana.jpg`

Reply:
The image has been uploaded.

Reply:
@mawy7 I don't see it - I only see the `www/config.yml` in this pull request. Did you commit it and update this pull request?

Reply:
sorry about that i didn't commit it to pyladies, i ran it locally and have commited
Thanks